doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -67,21 +67,45 @@ Note that each requirement may have one or more test cases associated with it.
 
 <a name="4.3.4"></a>
 ### 4.3.4 Interfaces & APIs
-This defines the test cases around the functionality that are exposed by OpenStack APIs. All the defined OpenStack 
-service APIs in RA-1's [chapter 05](../../../ref_arch/openstack/chapters/chapter05.md) will be the scope here. 
+This defines the test cases around the functionality that are exposed by OpenStack APIs. All the defined OpenStack
+service APIs in RA-1's [chapter 05](../../../ref_arch/openstack/chapters/chapter05.md) will be the scope here.
 
 Note: It will only target the functionality that are exposed by standard OpenStack APIs.  
 
 #### 4.3.4.1 Identity - Keystone
 
-It covers the test cases against identity management operations like user management, project management, multi-tenancy etc. 
-
-| Test case # | sub-category | Description | Requirement # |
-|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
-| `cert.test.vim.01` | API | Show API endpoint catalog. It *must* have endpoint for each Core service. | `req.int.api.01` |
+Keystone API is covered in the OpenStack Gates via
+[Tempest](https://opendev.org/openstack/tempest) and
+[keystone-tempest-plugin](https://opendev.org/openstack/keystone-tempest-plugin)
+as integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml).
+
+According to
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
+the next test names must not be executed:
+- .\*scenario.test_federated_authentication
+- .\*identity.admin.v2
+- .\*identity.v2
+
+Keystone API is also covered by [Rally](https://opendev.org/openstack/rally).
+
+Here are the mainline tasks integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml):
+- Authenticate.keystone
+- KeystoneBasic.add_and_remove_user_role
+- KeystoneBasic.create_add_and_list_user_roles
+- KeystoneBasic.create_and_list_tenants
```

Comment:
@collivier - can we create a table which traces these tasks to actual requirements?  The ask is to create TC Traceability to RM/RA requirements, so we need to list out TCs and their respective requirements, which also shows we're either testin exactly what has been asked, or if testing is going above and beyond what is needed.  A table mapping requirements and TCs will be great visual aid.

Reply:
@mf4716 You mean something like [this](https://github.com/cntt-n/CNTT/issues/830#issuecomment-568427423) ?

Reply:
@mf4716 @deebhatia 
You may be not aware about the OpenStack gates and how tempest is designed.
Here it's straightforward: it executes all tempest testcases except the optional capabilities in RA1 Chapter 5. 

It answers the simple requirements asked by RA1 Chapter 5: test Keystone ...
I can map the optional services and the tests not exected if it helps.

As answered in https://github.com/cntt-n/CNTT/issues/830#issuecomment-568427423, I don't see any technical reasons to duplicate the upstream work and then to list the ~2000 tempest tests already executed in Functest CNTT. It costs lots of time for a duplication in the best cases.
See opnfv/functest-smoke-cntt:hunter in https://build.opnfv.org/ci/view/functest/job/functest-hunter-daily/373/

Reply:
> @mf4716 You mean something like [this](https://github.com/cntt-n/CNTT/issues/830#issuecomment-568427423) ?

Yes @deebhatia - exactly like that.  For now, I'll approve the initial content in this PR, but we need to iterate on this for when we do our RC-1 alpha field trial.  For Snezka, its a start to have the requirements / cases listed, but we need traceability for certification and to normalize results.  Without it, we should not expect to receive a Certification Badge.  Approving.